src/azul/drs.py

```diff
@@ -75,19 +76,42 @@ class DRSClient:
 
     def get_object(self,
                    drs_uri: str,
-                   *,
-                   access_id: Optional[str] = None,
                    access_method: AccessMethod = AccessMethod.https
                    ) -> Access:
         """
         Returns access to the content of the data object identified by the
         given URI. The scheme of the URL in the returned access object depends
         on the access method specified.
         """
-        if access_id is None:
-            return self._get_object(drs_uri, access_method)
```

Comment:
I'd like to not inline _get_object at this time so as to keep the diff small and maintain symmetry.